src/haddock/libs/libhpc.py

```diff
@@ -21,9 +21,10 @@
     "COMPLETING": "running",
     "COMPLETED": "finished",
     "FAILED": "failed",
+    "TIMEOUT": "failed",
```

Comment:
Not sure if I understand this logic, why is the status of `TIMEOUT` not `timeout`?

Reply:
Because the only thing that is search in the script is the mapped value `failed`.

Reply:
Why not set is a timeout? Here you are overlapping states - a job might fail because of parameter issues, timeout is not related to that.